src/Microsoft.Data.SqlClient/tests/ManualTests/EventSource/EventSourceTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests.EventSource
+{
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp, "Not Implemented")]
+    public class EventSourceTest
+    {
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
+        public void IsTraceEnabled()
+        {
+            using (var listener = new SampleEventListener())
+            {
+                using (SqlConnection connection = new SqlConnection(DataTestUtility.TCPConnectionString))
+                {
+                    connection.Open();
+                    using (SqlCommand command = new SqlCommand("SELECT * From [Customers]", connection))
+                    {
+                        command.ExecuteNonQuery();
+                    }
+                }
+
+                //Check if all the events are from Trace
+                foreach (var item in listener.eventsNames)
+                {
+                    Assert.Contains("Trace", item);
+                }
+#if net46
+                //Check if we can disable the events
+                listener.DisableEvents(SqlClientEventSource.Log);
```

Comment:
How are you able to access `SqlClientEventSource` in the tests? You have made this internal. 
Are we exposing internals to the tests ? 
Also can you add a test with the event ID string, instead of the strong types?

Reply:
I will change this while I am working in NetCore. We had an internal reference to source code from tests, but it is changed. Tests will be more extensive when EventSource is added to NetCore.

Reply:
Fixed.